requirements/base.in

```diff
@@ -4,3 +4,4 @@
 Django             # Web application framework
 django-model-utils        # Provides TimeStampedModel abstract base class
 git+https://github.com/open-craft/openedx-events.git@77ad965c4ac5157b861a19b8cb6b9240883008ba#egg=openedx_events
+git+https://github.com/open-craft/openedx-filters.git@d0206cf84f2e5e22b1a99d06566d88839d40b9fe#egg=openedx-filters
```

Comment:
Nit: Please add a TODO here to update these requirements once the corresponding PRs are merged upstream.